hicommonwealth/commonwealth
# Refactor New Snapshot Proposal to React

Refactors the New Snapshot Proposal page from Mithril to React, including the quill editor component.

## Link to Issue
Closes: #3259

## Description of Changes
- Refactors New Snapshot Proposal Page
- Refactor the quill component on the page

## Test Plan
- Navigate to  http://localhost:8080/dydx/new/snapshot/dydxgov.eth
- On the New Snapshot Proposal page– confirm that the quill component renders and works
- Fill the form and submit– confirm that it creates the proposal

## Deployment Plan
N/A

## Other Considerations
N/A
